static/bidder-info/inmobi.yaml

```diff
@@ -17,4 +17,6 @@ userSync:
   iframe:
     url: "https://sync.inmobi.com/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
     userMacro: "{ID5UID}"
-
+  redirect:
+    url: "https://sync.inmobi.com/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
+    userMacro: "{ID5UID}"
```

Comment:
Verified user sync redirect is working correctly and inmobi user ID is correctly added to cookie

<img width="1174" alt="Screenshot 2025-04-24 at 11 36 32 AM" src="https://github.com/user-attachments/assets/de962eae-3452-49bd-b0e7-ad06ce5cb321" />